netflix/src/components/home/Wrapper.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+import { useEffect, useState } from 'react';
+import axios from '@/api/axios';
+import request from '@/api/request';
+import Image from 'next/image';
+
+interface WrapperProps {
+  title: string;
+  fetchType: string; // 다른 종류의 영화 목록을 불러오기 위해 fetchType 추가
+}
+
+interface Movie {
+  id: number;
+  poster_path: string;
+  title: string;
+}
+
+export default function Wrapper(props: WrapperProps) {
+  const { title, fetchType } = props;
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  useEffect(() => {
+    async function fetchMovies() {
+      try {
+        let url = request.fetchPopular; // 기본값 설정
+        if (fetchType === 'trending') {
+          url = request.fetchTrending;
+        } else if (fetchType === 'topRated') {
+          url = request.fetchTopRated;
+        } else if (fetchType === 'horrorMovies') {
+          url = request.fetchHorrorMovies;
+        }
```

Comment:
```suggestion
const urlMap = {
  trending: request.fetchTrending,
  topRated: request.fetchTopRated,
  horrorMovies: request.fetchHorrorMovies,
  default: request.fetchPopular,
};

const url = urlMap[fetchType] || urlMap.default;
```

URL 결정 로직도 이런식으로 개선하면 더 좋은 코드가 될 것 같아요